### PR TITLE
Stores and loads additional filter in query params

### PR DESCRIPTION
**Pull request by MTrost**
Tasks
- [x] Somehow indicate that filter is active
- [x] Make filter removable
- [x] Try to apply additional filters from query params from the start. atm it fetches everything first and then it refetches/renders with filters applied

### CODE REVIEW

**Comment by MTrost** [comment]
- [ ] Somehow indicate that filter is active
- [ ] Make filter removable

File: code/test/e2e/loggedin/filter_wizard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  await page.pause();
+  await page.getByRole('button', { name: 'Filter' }).click();
+  await page.getByText('attribute name').click();
+  await page.getByRole('option', { name: 'name' }).click();
+  await page.locator('div[role="alert"]:has-text("operation")').click();
+  await page.getByRole('option', { name: 'Equal' }).getByText('Equal').click();
+  await page.getByText('value').click();
+  await page.locator('span:has-text("e2e-Test-Do_not_delete")').click();
+  await page.getByRole('button', { name: 'Done' }).click();
+  await page.waitForURL('https://nui.localhost/ui/edges?view=table&nuvlabox=name=%27e2e-Test-Do_not_delete%27');
```
Comment (0xbase12) [inline code/test/e2e/loggedin/filter_wizard.spec.ts:20]: e2e test failing and hardcoded urls

Comment (0xbase12) [reply code/test/e2e/loggedin/filter_wizard.spec.ts:20]: Will send a video with bugs:
- popup visible when modal is open
- remove filter doesn't work properly on edge page

**Review by 0xbase12 — CHANGES_REQUESTED** [review]

File: code/test/e2e/loggedin/filter_wizard.spec.ts (hunk 1 above)
Comment (MTrost) [reply code/test/e2e/loggedin/filter_wizard.spec.ts]: Fixed both plus the e2e test URLs

**Review by 0xbase12 — APPROVED** [review]

**Pull request merged**